nemo/utils/exp_manager.py

```diff
@@ -402,6 +402,12 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
         # extend training loop to skip initial validation when resuming from checkpoint
         configure_no_restart_validation_training_loop(trainer)
 
+    # Setup a stateless timer for use on clusters.
+    if cfg.max_time is not None:
```

Comment:
Need to add this key to the exp manager config.

Reply:
Done!